src/recursive_sorting/example.py

```diff
@@ -0,0 +1,42 @@
+def merge_sort(sort_list):
```

Comment:
Cool solution! Your merge functionality is doing some extra work. The merge can be done with a single loop with some if/elif statement. Think about how you can change the first while loop so that you don't need the other two.